contents/java/Spring.md

```diff
@@ -0,0 +1,193 @@
+<frontmatter>
+  title: Introduction to Spring Framework
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Spring Framework
+
+**Authors: [Liu Yiwen](https://github.com/0blivious)** <br>
+Reviewers: []()
+
+<box type="info">
+This chapter assumes that the reader has a basic knowledge of Java enterprise.
+</box>
+
+<box id="article-toc">
+
+* [What is Spring Framework‎](#what-is-spring-framework)
+* [Core Features](#core-features)
+  * [The IOC Container](#the-ioc-container)
+  * [Spring JDBCTemplate](#spring-jdbctemplate)
+  * [Spring AOP](#spring-aop)
+* [Why use Spring‎](#why-use-spring)
```

Comment:
The benefits of Spring are not included in the ToC. Although each section is short, maybe it is better to include them for consistency.